packages/client/modules/teamDashboard/components/TeamDashInsights/MeetingEngagementCard.tsx

```diff
@@ -0,0 +1,62 @@
+import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import {MeetingEngagementCard_insights$key} from '~/__generated__/MeetingEngagementCard_insights.graphql'
+import TeamInsightsCard from './TeamInsightsCard'
+
+// We need to tune this once we have some data on real teams
+const HIGH_ENGAGEMENT_THRESHOLD = 0.7
+const LOW_ENGAGEMENT_THRESHOLD = 0.3
+
+const getDescription = (engagement: number) => {
+  if (engagement >= HIGH_ENGAGEMENT_THRESHOLD) {
+    return 'Great job! Your meetings have a higher engagement rate than the average team using Parabol. Keep it up!'
+  }
+  if (engagement <= LOW_ENGAGEMENT_THRESHOLD) {
+    return 'Your engagement rate is lower than other Parabol teams. Try starting with an icebreaker to get everyone talking or discuss why people don’t feel like contributing to your meeting.'
+  }
+  return 'You have an average meeting engagement rate, but there’s always room for improvement! Try mixing things up with a new template.'
+}
```

Comment:
@acressall I'm not sure about these copies, I just came up with the average one.

Reply:
The idea was it would be based on what the % result is. So we could say 0-33% is low; 34-66% is average; 67-100% is high?

Reply:
Ok, are you suggesting we don't update the message until we've seen the engagement score in action for a bit?